### PR TITLE
CONTRIBUTING: Fix syntax and remove repeating content

### PR DESCRIPTION
**Pull request by prtksxna**
Not sure how but #72 added some repeated content and in HTML
instead of MD.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,34 +29,29 @@ $ npm run build
...
 
-<h3>Get the design repository</h3>
-						<p>The <a href="https://github.com/wikimedia/WikimediaUI-Style-Guide/">WikimediaUI Style Guide repository is available to download</a>. It contains this documentation, Sketch and SVG templates and resources. Select the “clone or download” button to get the repository.</p>
+### Get the design repository
+The [WikimediaUI Style Guide repository is available to download](https://github.com/wikimedia/WikimediaUI-Style-Guide/). It contains this documentation, Sketch and SVG templates and resources. Select the “clone or download” button to get the repository.
+
+### Commit changes
+Easily contribute to the content of the style guide by adding new recommendations, making corrections or adding examples in the form of images or videos:
+
+If you have <a href="https://en.wikipedia.org/wiki/Git">Git</a> installed on your system, <a href="https://help.github.com/articles/cloning-a-repository/">cloning</a> is preferred way to get the contents since it allows you to contribute back. Technically changes can be submitted as usual in Git.
```
Comment (Volker-E) [inline CONTRIBUTING.md]: - Should be markdown links too
- The sentence “Technically changes…” now seems more noise to me than actual helpful content. Let's remove it…?!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,34 +29,29 @@ $ npm run build
...
-						We're using <a href="http://stylelint.io/">stylelint</a> to ensure the CSS <a href="https://github.com/wikimedia/stylelint-config-wikimedia/">aligns with our</a> <a href="https://www.mediawiki.org/wiki/Manual:Coding_conventions/CSS">coding conventions</a>.</p>
-						<p>These requirements provide our contributors a reliable environment and our viewers a performant experience.</p>
+When modifying aspects such as the CSS styling you'll need to rebuild the documentation files using [Grunt](http://gruntjs.com/) (which requires [Node.js and npm](https://docs.npmjs.com/getting-started/installing-node)). The first time you have to run `npm install` in a terminal from the style guide's main folder in order to get all necessary dependencies. Every time you want to rebuild the CSS files for the documentation you just need to launch the `grunt` command in a terminal, also from the main folder.
```
Comment (Volker-E) [inline CONTRIBUTING.md]: - Would recommend “…rebuild the production files”
- https://gruntjs.com/
- Would recommend “To rebuild the CSS files after changing them in wmui-style-guide.dev.css you just need to launch the `grunt` command in a terminal, also from the main folder.”

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,34 +29,29 @@ $ npm run build
...
 
-						<p>You can participate in many ways. The documentation is available in a Git repository with the design assets needed for your project. You can get all of them, change anything and send the changes back.</p>
+We're using [stylelint](http://stylelint.io/) to ensure the CSS [aligns with our](https://github.com/wikimedia/stylelint-config-wikimedia/) [coding conventions](https://www.mediawiki.org/wiki/Manual:Coding_conventions/CSS).
```
Comment (Volker-E) [inline CONTRIBUTING.md]: - https://stylelint.io

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -29,34 +29,29 @@ $ npm run build
 
 Please add specific topics into a single commit and also take into account the [Wikimedia commit message guidelines](https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines).
 
-Push to your fork and [submit a pull request][pr]
-
-[pr]: https://github.com/wikimedia/WikimediaUI-Style-Guide/compare
+Push to your fork and [submit a pull request](https://github.com/wikimedia/WikimediaUI-Style-Guide/compare)
```
Comment (Volker-E) [inline CONTRIBUTING.md]: This and the next paragraph should be underneath Commit changes.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -29,34 +29,29 @@ $ npm run build
...
 
 In order for the pushed changes to be available live, members of Wikimedia design are going to review it first.
 
-<h3>Get the design repository</h3>
-						<p>The <a href="https://github.com/wikimedia/WikimediaUI-Style-Guide/">WikimediaUI Style Guide repository is available to download</a>. It contains this documentation, Sketch and SVG templates and resources. Select the “clone or download” button to get the repository.</p>
+### Get the design repository
```
Comment (Volker-E) [inline CONTRIBUTING.md]: We should go for higher-level heading. h1 or h2. Leaving it up to you for what feels more sensible.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -29,34 +29,29 @@ $ npm run build
...
 
-						<h3>Commit changes</h3>
-						<p>Easily contribute to the content of the style guide by adding new recommendations, making corrections or adding examples in the form of images or videos: </p>
-						<p>If you have <a href="https://en.wikipedia.org/wiki/Git">Git</a> installed on your system, <a href="https://help.github.com/articles/cloning-a-repository/">cloning</a> is preferred way to get the contents since it allows you to contribute back. Technically changes can be submitted as usual in Git.</p>
-						<p>In order to publish the updated contents to be available live, go to the repo and launch the following command from a terminal: <code>git push -f origin master:gh-pages</code></p>
-						<p>Please add specific topics into a single commit and also take into account the <a href="https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines" target="_blank">Wikimedia commit message guidelines</a>.</p>
+You can participate in many ways. The documentation is available in a Git repository with the design assets needed for your project. You can get all of them, change anything and send the changes back.
```
Comment (Volker-E) [inline CONTRIBUTING.md]: This should be at top of the file, probably the starting paragraph?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -29,34 +29,29 @@ $ npm run build
 
 Please add specific topics into a single commit and also take into account the [Wikimedia commit message guidelines](https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines).
```
Comment (Volker-E) [inline CONTRIBUTING.md]: Duplicated with content below underneath commit changes. Let's remove it

File: CONTRIBUTING.md (hunk 1 above)
Comment (prtksxna) [reply CONTRIBUTING.md]: Done

**Comment by prtksxna** [comment]
Updated with changes :)

**Comment by Volker-E** [comment]
Thanks, @prtksxna!

**Pull request merged**